### PR TITLE
Added Haskell Language support

### LINKED ISSUE #6: add grammar for haskell
see [contributing instructions](https://github.com/asottile/babi-grammars#internalcontributing-instructions) if you'd like to work on this

### PR DESCRIPTION
**Pull request by ClasherKasten**
Closes  #6

### CODE REVIEW

File: bin/test-grammars [hunk 1]
```diff
@@ -33,13 +33,15 @@ TODO_TEST = frozenset((
     'source.tsx',
     'source.x86',
     'source.x86_64',
+    'source.haskell',
```
Comment (asottile) [inline bin/test-grammars:36]: new grammars should have tests

Comment (ClasherKasten) [reply bin/test-grammars:36]: Oh, sorry, i didn't had understand it in this moment. Going to fix it.

Comment (ClasherKasten) [reply bin/test-grammars:36]: I tried to added `testdata/source_haskell.hs` as a testcase, I update the syntax, I download the syntax, I do `pip install . babi`, but i always get the following result:
babi_haskell

Comment (asottile) [reply bin/test-grammars:36]: do you have a theme installed?

Comment (ClasherKasten) [reply bin/test-grammars:36]: yeah, the standard vs-code-dark theme. Normally it look like this
babi_normal
:

Comment (asottile) [reply bin/test-grammars:36]: `hash -r`? maybe it's using another `babi` on your system?

Comment (ClasherKasten) [reply bin/test-grammars:36]: Yep, was the false babi, it had used the global installed babi, when it should have used the babi installed in the virtualenv. Thanks for helping me with this,

**Review by asottile — APPROVED** [review]

**Pull request merged**